lib/dry/operation/class_context/prepender.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Dry
+  class Operation
+    module ClassContext
+      # @api private
+      class Prepender < Module
```

Comment:
I renamed it to `MethodPrepender`. It's not perfect yet, but I'd like to avoid using different naming conventions like `MethodDecorator` and consistently use "prepend" everywhere (except for the `operate_on` for ergonomic reasons). It's also more explicit. Do you have a better idea? :thinking:

Reply:
Yep, `MethodPrepender` is an improvement! But I wonder if we can make it even clearer. This isn't just prepending random behaviour on any old methods; it's wrapping those methods in an implicit `steps` block. So maybe `StepsMethodPrepender`?

This could also help us discover a name for these methods that we use as we describe the gem to our users... are these, in fact, "steps methods?"

Reply:
This makes me think we're probably deviating from the agreed naming convention on the `#operates_on` method. Should we call it `#steps_on`? We could also rename `#skip_prepending` to `#skip_steps_prepending`. What do you think, @timriley?